scripts/insert_into_appointments.sql

```diff
@@ -1 +1,11 @@
 -- insert appointments creating connections between doctors and patients
+
+
+INSERT INTO appointments(date, time, comment, doctor_id, patient_id)
+  VALUES ('2016-08-12', '10:00', 5, 9),
+         ('2016-08-17', '13:00', 7, 6),
+         ('2016-08-12', '10:00', 7, 9),
+         ('2016-08-17', '13:00', 5, 8)
+;
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables.